sbt-mode-comint.el

```diff
@@ -185,15 +185,14 @@ what `sbt:move-marker-before-prompt-filter` did."
 (defun sbt:scala-escape-string (str)
   (mapconcat 'sbt:scala-escape-char str ""))
 
-(defconst sbt:completions-regex "^\\[completions\\] \\(.*\\)
?$")
+(defconst sbt:completions-regex "^\\[completions\\] \\(.*?\\)
?$")
```

Comment:
that weird control character is still here

Reply:
that's an optional newline I guess?